streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsAssignmentScaleTest.java

```diff
@@ -190,11 +197,35 @@ private void completeLargeAssignment(final int numPartitions,
         configMap.put(InternalConfig.INTERNAL_TASK_ASSIGNOR_CLASS, taskAssignor.getName());
         configMap.put(StreamsConfig.NUM_STANDBY_REPLICAS_CONFIG, numStandbys);
 
-        final MockInternalTopicManager mockInternalTopicManager = new MockInternalTopicManager(
+        configMap.put(StreamsConfig.RACK_AWARE_ASSIGNMENT_STRATEGY_CONFIG,
+            StreamsConfig.RACK_AWARE_ASSIGNMENT_STRATEGY_NONE);
```

Comment:
I forget, can we run it with the other rack-aware strategy? Or did both of them take too long for this scale test?

Reply:
No they both take way too long for some of the scale tests.